pkg/operator/operator.go

```diff
@@ -80,7 +78,8 @@ func NewOperator() (context.Context, *Operator) {
 	ctx = knativeinjection.WithNamespaceScope(ctx, system.Namespace())
 
 	// Options
-	opts := lo.Must(options.New().Parse(os.Args[1:]...))
+	ctx = injection.WithOptionsOrDie(ctx, apis.Options...)
+	opts := options.FromContext(ctx)
```

Comment:
Do we need to store the opts like this if they are already injected into the context? We can just pull them out of the context starting from here, right?

Reply:
We can, its used a lot though so there's some value in taking it out here for brevity elsewhere. Definitely a style preference though, I can go either way.

Reply:
I don't care either way either. Just a question. I'm fine either way.